files/build_templates/sonic_debian_extension.j2

```diff
@@ -255,7 +255,11 @@ sudo cp -f $IMAGE_CONFIGS/bash/bash.bashrc $FILESYSTEM_ROOT/etc/
 
 # Install prerequisites needed for installing the dependent Python packages of sonic-host-services
 # These packages can be uninstalled after installation
-sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install libcairo2-dev libdbus-1-dev libgirepository1.0-dev libsystemd-dev pkg-config
+if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
+    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install libcairo2-dev libdbus-1-dev libgirepository1.0-dev libsystemd-dev=247.3-3~bpo10+1 pkg-config
```

Comment:
why specify the version here for libsystemd-dev?

Reply:
This change is for the error "libsystemd-dev : Depends: libsystemd0 (= 241-7~deb10u7) but 247.3-3~bpo10+1 is to be installed" when install libsystemd-dev. It should be related to PR #7228,  version of  libsystemd0 have be upgraded to 247 during apt update.   libsystemd-dev's default version should be 241 for arm64 and it depends on libsystemd0 241. ARM64 and AMD64 doesn't have the same apt source list. The change is really not good, do you have good suggestion, thanks.

Reply:
i can see it here. https://packages.debian.org/buster-backports/libsystemd-dev

Reply:
247 is trying to fix a bug, i think we need that.

Reply:
It seems that default version for libsystemd-dev is 241-7 ~ deb10u7 for ARM arch.  The version of libsystemd-dev in buster-main(https://packages.debian.org/buster/libsystemd-dev) is 241-7 ~ deb10u7, while version in buster-backports is 247.3-3~bpo10+1 (https://packages.debian.org/buster-backports/libsystemd-dev).  

 Is "apt-get -y -t $IMAGE_DISTRO-backports" acceptable here? Just like following:
-sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install libcairo2-dev libdbus-1-dev libgirepository1.0-dev libsystemd-dev pkg-config
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install libcairo2-dev libdbus-1-dev libgirepository1.0-dev pkg-config
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y -t $IMAGE_DISTRO-backports install libsystemd-dev

Reply:
okay for me